patchwork/steps/GetTypescriptTypeInfo/GetTypescriptTypeInfo.py

```diff
@@ -0,0 +1,45 @@
+import os
+import subprocess
+from pathlib import Path
+
+from patchwork.step import Step
+
+
+_DEFAULT_TS_FILE = Path(__file__).parent / "get_type_info.ts"
+
+
+class GetTypescriptTypeInfo(Step):
```

Comment:
```
class PRPB(Step, input_class=PRPBInputs, output_class=PRPBOutputs):
```

Can the input and output class be linked to the step itself. Thanks. Once this is done the key check will be done in the super constructor instead.